pkg/printer/package.go

```diff
@@ -0,0 +1,50 @@
+package printer
+
+import (
+	"fmt"
+	"github.com/cnoe-io/idpbuilder/pkg/types"
+	"io"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type PackagePrinter struct {
+	Packages  []types.Package
+	OutWriter io.Writer
+}
+
+func (pp PackagePrinter) PrintOutput(format string) error {
+	switch format {
+	case "json":
+		return PrintDataAsJson(pp.Packages, pp.OutWriter)
+	case "yaml":
+		return PrintDataAsYaml(pp.Packages, pp.OutWriter)
+	case "table":
+		return PrintDataAsTable(generatePackageTable(pp.Packages), pp.OutWriter)
+	default:
+		return fmt.Errorf("output format %s is not supported", format)
+	}
+}
+
+func generatePackageTable(packagesTable []types.Package) metav1.Table {
+	table := &metav1.Table{}
+	table.ColumnDefinitions = []metav1.TableColumnDefinition{
+		{Name: "Custom package name", Type: "string"},
```

Comment:
I liked "name" more. Custom package is already implied when you run the command.

Reply:
Fixed: 9bbd4d5